mesonbuild/dependencies/configtool.py

```diff
@@ -69,6 +69,21 @@ def _sanitize_version(self, version: str) -> str:
             return m.group(0).rstrip('.')
         return version
 
+    def _check_and_get_version(self, tool, returncode):
```

Comment:
CI fails because you added a function but it doesn't have type annotations.

Reply:
Annotations added so that the CI is now happy.